DependencyInjection/SonataDoctrinePHPCRAdminExtension.php

```diff
@@ -83,79 +88,56 @@ public function getNamespace()
     }
 
     /**
-     * Set the tree type mapping configuration in the services.
-     *
-     * @param array            $config
-     * @param ContainerBuilder $container
+     * {@inheritdoc}
      */
-    private function loadTreeTypes($config, ContainerBuilder $container)
+    public function prepend(ContainerBuilder $container)
```

Comment:
When `repository_name` for our configuration is null, we have to set the default one of `cmf_resourse`. We can also do the default setting in the controller also, but we can not use `null`at all?

So we have 3 possibilities from my POV:
- fetch the default value from `resource-bundle`
- force the user to set it on its own – would do it twice now
- pass the default value of `resource-bundle` to our controller, and use that in case of a `null`

Do you have an other idea?